src/ring_theory/polynomial/cyclotomic.lean

```diff
@@ -495,6 +500,51 @@ begin
   rwa [← mem_primitive_roots hpos] at h,
 end
 
+private lemma is_root_cyclotomic_iff' {n : ℕ} {K : Type*} [field K] {μ : K} (hn : (n : K) ≠ 0) :
+  is_primitive_root μ n ↔ is_root (cyclotomic n K) μ :=
```

Comment:
The name and the statement don't agree about should be the LHS and the RHS of the iff.